aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -281,10 +267,43 @@ index_select_add(
                   select_indices_data + offsets_data[start_idx]);
             }
           });
-      return;
+    } else {
+      using bfloat16 = int16_t;
```

Comment:
seems the code is very like the float16 path, can we reuse?

Reply:
We have to use uint16_t as fbgemm::bfloat16, and add an arg `isbf16` to distinguish fp16/bf16. Now the code here is reused better.